saos-webapp/src/main/java/pl/edu/icm/saos/webapp/judgment/detail/correction/JudgmentCorrectionService.java

```diff
@@ -0,0 +1,105 @@
+package pl.edu.icm.saos.webapp.judgment.detail.correction;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import pl.edu.icm.saos.persistence.correction.JudgmentCorrectionRepository;
+import pl.edu.icm.saos.persistence.correction.model.ChangeOperation;
+import pl.edu.icm.saos.persistence.correction.model.JudgmentCorrection;
+import pl.edu.icm.saos.persistence.model.CommonCourtJudgment;
+import pl.edu.icm.saos.persistence.model.Judge;
+import pl.edu.icm.saos.persistence.model.SupremeCourtChamber;
+import pl.edu.icm.saos.persistence.model.SupremeCourtJudgment;
+import pl.edu.icm.saos.persistence.model.SupremeCourtJudgmentForm;
+
+
+
+/**
+ * 
+ * @author Łukasz Pawełczak
+ *
+ */
+@Service
+public class JudgmentCorrectionService {
+
+	
+	private JudgmentCorrectionRepository judgmentCorrectionRepository;
+	
+	
+	//------------------------ LOGIC --------------------------
+	
+	public List<JudgmentCorrection> findByJugmentId(int judgmentId) {
```

Comment:
Maybe the method name should say a little more for example: findAllByJudgmentIdSorted or something like this.